packages/vuetify/src/components/VBtn/VBtn.tsx

```diff
@@ -110,7 +117,7 @@ export const VBtn = defineComponent({
 
     useRender(() => {
       const Tag = (link.isLink.value) ? 'a' : props.tag
-      const hasColor = !group || group.isSelected.value
+      const hasColor = !group || (isActive.value && group.isSelected.value)
```

Comment:
```suggestion
      const hasColor = !group || isActive.value
```

Reply:
isActive already takes group.isSelected into account

Reply:
Thanks! feedback is addressed